### ISSUE TITLE
CMS source of truth for Mental Phone Numbers at facilities

### ISSUE THREAD

**Issue by kevwalsh** [department-of-veterans-affairs/va.gov-cms#9717]
## Background
* As of July 2023, MH phone number data is maintained in a hand-curated CSV owned by a VA employee. This data is the source of truth for Lighthouse. This CSV will be replaced / deprecated.
* As of July 2023, the CMS receives mental health phone numbers from lighthouse Facilities API. Moving forward, we want CMS to be the source of truth and to send definitive MH numbers to the Facilities API / Lighthouse.
* Legislative mandates guide the business requirements for this change (below)

**VHA DM sync**: related Trello card: https://trello.com/c/lvqPRuCI/33-mental-health-phone-numbers
**Lighthouse sync**: related Trello card: https://trello.com/c/x8YdixQ6/6-mental-health-phone-numbers

### Legislative mandate
(b) Elements.--The Internet website published under subsection (a) shall provide to veterans information regarding all of the mental health care services available in the Veteran Integrated Service Network that the veteran is seeking such services, including, with respect to each medical center, Vet Center (as defined in section 1712A of title 38, United States Code), and community-based outpatient center in the Veterans Integrated Service Network--           
    (1) the name and contact information of each social work office;            
    (2) the name and contact information of each mental health clinic;            
    (3) a list of appropriate staff; and            
    (4) any other information the Secretary determines appropriate.    
(c) <<NOTE: Deadline.>>  Updated Information.--The Secretary shall ensure that the information described in subsection  (b) that is published on the Internet website under subsection  (a) is updated not less than **once every 90 days**.

https://www.congress.gov/bill/114th-congress/house-bill/203/text

## User Story or Problem Statement

As a Veteran, I need know with confidence what Mental Health phone number is available in my facilities so I can get mental health support.

As a VAMC Editor, I need to know where to record Mental Health phone numbers so that I can provide accurate help info to Veterans. 

As VHA DM, I need definitive data handling for Mental health phone numbers, so I can direct Editors to right way to govern this data. (?)

As Facilities team, I need a clear data map of where Mental Health phone numbers are maintained, so we can provide accurate single source of truth data to Veterans.

## Affected users and stakeholders

* VHA DM
* CMS editors of VAMCs
* Lighthouse, potentially / probably

## Hypothesis
We believe that maintaining mental health phone numbers in the CMS, rather than in VAST or an external CSV, will be the most robust and maintainable way to ensure accurate data is provided to veterans.

## Assumptions
* Re: Lighthouse V0 vs. V1: Mental health phone numbers already exist in V0 so there will be no change in functionality there between V0/V1; when we are ready to migrate to CMS as the source of MH numbers, that new sourcing will apply to both versions. [Slack thread](https://dsva.slack.com/archives/C02BTJTDFTN/p1690829674767869?thread_ts=1690820215.487199&cid=C02BTJTDFTN)

## Acceptance Criteria
- [ ] 

### Change management triage

The change represented by this user story will:
- [ ] Be a site-wide change to appearance or key functionality (such as log-in process)
- [ ] Be a specific change that requires more than 30 minutes of work by a user to meet a publishing requirement
- [ ] Remove a piece of functionality (such as restricting editing functions, or “hardening”)
- [x] Require action by some or all users by a specific deadline
- [ ] Change the front end and be highly visible to the public
- [ ] Noticeable performance improvements (publishing speed, predictability)

If you selected an item above, open a new issue using the [change management template](https://github.com/department-of-veterans-affairs/va.gov-cms/blob/main/.github/ISSUE_TEMPLATE/change-management.md).

If you did not select an item above, update issues/PRs in GitHub but don’t plan for change management.

**Comment by jilladams** [comment]
* Fleshed out body of the epic with notes from today's VHA DM sync on the background / legislative requirement
* Removed the Lighthouse V1 blocker. Adam confirmed today that we are not blocked on V1, MH prep is present in V0 & V1 both already, and when we are ready to cut over to CMS as source of truth, it'll be ready in both. 
* Drafted user stories. They could use a onceover.

**Comment by jilladams** [comment]
Note from VHA DM sync: current state of the CSV ownership, and where we want MH numbers to reside in CMS terms, are open questions. https://trello.com/c/lvqPRuCI/33-mental-health-phone-numbers-1-6

* Mental Health phone number required at the System level.
    * We want to confirm this is our proposal before we take that to Dr. Jay Cohen & his Office of Mental Health & Suicide Prevention (OMHSP).
    * VHA DM has a comms colleague, Bobbi Hauptmann, who liases with OMHSP. Before we take anything to OMHSP, invite her here for review of proposal.
* **Governance**: Who is responsible for entering that number on the System? (is this data migration level, is it entered in CMS / by whom?, what do we do with existing numbers?)
    * A data map would help to spell out end state / help answer governance questions. Timing / priority: mental health risk is high enough to prioritize this from VHA DM perspective.
    * Update every 90 days: legal requirement

Next steps: Helpful content deliverable: documentation of the current state of phone number sources of truth vs. desired end state. ([#14578](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14578))

**Comment by jilladams** [comment]
Noting also: Governance isn't ticketed, but we may decide we need to do that.

**Comment by jilladams** [comment]
Notes on Vet Center implications: https://dsva.slack.com/archives/C0FQSS30V/p1690902017945119?thread_ts=1690835147.244549&cid=C0FQSS30V

Notes on "health service validation process"  unrelated to this effort, but noting: that's the case: https://dsva.slack.com/archives/C0FQSS30V/p1690902017945119?thread_ts=1690835147.244549&cid=C0FQSS30V

**Comment by swirtSJW** [comment]
The push part was moved out into its own PR so we could safely seed LH and have the data persist.   It is only pushing the Mental health phone we already have and is not adding the new field.

**Comment by davidmpickett** [comment]
@Agile6MSkinner My recommendation would be to move remaining tickets into #16590 and then close this epic

**Comment by Agile6MSkinner** [comment]
Done
